sim/core/sim.go

```diff
@@ -511,6 +512,9 @@ func (sim *Simulation) advanceWeaponAttacks() {
 
 	sim.minWeaponAttackTime = NeverExpires
 	for _, wa := range sim.weaponAttacks {
+		/*if sim.Log != nil {
```

Comment:
Could you clean up this comment if you don't want to add this log?

Reply:
cleaned up